src/shared/ui/backgrounds/TicketOptionLayout.tsx

```
@@ -0,0 +1,95 @@
+    const pageIndicator = Array(ticketAmount).fill(" . ");
+    pageIndicator[currentPage - 1] = " - ";
+
+    //버튼 텍스트
+    const isLastPage = currentPage === ticketAmount;
+    const buttonText = isLastPage ? "결제하기" : "다음 티켓 옵션 선택하기";
+    const handleNextPage = () => {
+        if (isLastPage) {
+            {/* 데이터 전송 추가 */}
+        } else {
+            setCurrentPage(currentPage + 1);
+            {/* 데이터 전송 추가 */}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)